app/models/course/assessment/category.rb

```diff
@@ -8,10 +10,16 @@ class Course::Assessment::Category < ActiveRecord::Base
 
   accepts_nested_attributes_for :tabs
 
+  before_create :build_initial_tab
   before_destroy :validate_before_destroy
 
   default_scope { order(:weight) }
 
+  def self.after_course_create(course)
+    course.assessment_categories.
+      create(title: Course::Assessment::Category.human_attribute_name('title.default'), weight: 0)
```

Comment:
Hmm, not sure I understand. `title.default` ends up pointing to the assessment header (used in places like breadcrumbs, assessment settings), which would be translated for a Chinese user. Do you mean we should skip going through ActiveRecord and use the key directly?